docs/guide/execute-transaction.md

```diff
@@ -95,21 +95,21 @@ Examples of parameter [`ExecParams`](https://algobuilder.dev/api/algob/modules/r
     payFlags: {}
   }
 ```
-- To learn about more parameters like (account, appArgs, ForeignApps, ForeignAssets etc).Please check [SSCOptionalFlags](https://algobuilder.dev/api/algob/interfaces/runtime.types.sscoptionalflags.html)
+- To learn about more parameters like (account, appArgs, ForeignApps, ForeignAssets etc).Please check [AppOptionalFlags](https://algobuilder.dev/api/algob/interfaces/runtime.types.AppOptionalFlags.html)
 
-### [Opt-In to SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#optinsscparam)
+### [Opt-In to SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#OptInAppparam)
 
 ```js
   {
-    type: TransactionType.OptInSSC,
+    type: TransactionType.OptInApp,
```

Comment:
Maybe we should rename to `OptInToApp` ? sounds better & similar to `OptInToASA`

Reply:
yes